pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto

```diff
@@ -1223,6 +1223,8 @@ message ContainerStats {
     MemoryUsage memory = 3;
     // Usage of the writeable layer.
     FilesystemUsage writable_layer = 4;
+    // Usage of the network interfaces.
```

Comment:
I may have missed the previous discussions. Why do we need per-container (as opposed to per-pod) network stats?

Reply:
See https://github.com/kubernetes/kubernetes/pull/67884#issuecomment-418580789. It is used to get network stats for Windows containers from runtime (e.g. docker).